torch/nn/functional.py

```diff
@@ -3833,6 +3841,12 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
         return torch._C._nn.upsample_linear1d(input, output_size, align_corners, scale_factors)
     if input.dim() == 4 and mode == "bilinear":
         assert align_corners is not None
+        # Enforce that the full call with the new kwarg is not invoked when scripting.
+        # TODO: Remove this scripting logic once the 2-week FC window has passed.
+        if antialias and torch.jit.is_scripting():
+            raise RuntimeError("TorchScript currently does not support antialias in interpolate")
+        if antialias:
```

Comment:
This branch needs to be gated by `not torch.jit.is_scripting()` as well. Note that scripting doesn't operate linearly - it will include all reachable code branches into the serialized result.

To achieve the goal of leaving out the call to `torch._C._nn._upsample_bilinear2d_aa` from the serialized JIT model, I'd suggest something like this:
```python
if antialias:
    if not torch.jit_scripting():
        return torch._C._nn._upsample_bilinear2d_aa(input, output_size, align_corners, scale_factors)
    else:
        raise RuntimeError("TorchScript currently does not support antialias in interpolate")
```